data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1843 @@
+				decline
+
+mission "Heliarch Recon Lagrange"
+	invisible
+	landing
+	source "Lagrange"
+	destination "Earth"
+	to offer
+		has "Heliarch Recon 1: active"
+	on offer
+		conversation
+			`You don't really know what the Heliarchs did to your sensors, or if you can even see the data they are collecting, but as you come in for a landing, you hear a faint whistling sound from the hull of your ship. It's short lived, and you land normally. Hopefully nothing broke.`
+				decline
+
+event "heliarch recon break"
+
+mission "Heliarch Recon 2-A"
+	name "Scanning Preparations"
+	description "Install an Outfit Scanner, then land on <destination>, where the Heliarch will modify it to be able to scan Quarg ships the way they need it."
+	landing
+	source
+		near "Ekuarik" 1 100
+	destination "Ring of Wisdom"
+	to offer
+		has "event: heliarch recon break"
+		not "Resistance: Questions: done"
+	on offer
+		conversation
+			`When you land, you're contacted by the Heliarch agents who had you land on some Quarg worlds and rings to collect data. "Captain <last>, a new task for you, we have," the Kimek says.`
+			choice
+				`	"What do you need me to do?"`
+				`	"I'm sorry, but I'm not willing to risk my ship any longer to help you spy on the Quarg."`
+					decline
+			`	"Too large and complex for a quick reading, planets and ringworlds are, so wish to get updated scans on Quarg ships, the consul now does," the Arach says.`
+			`	"Scan them during our war, we of course did, but much more precise now, our technology has become," the Saryd explains. "To use one of our own scanners, we cannot ask of you. Much risk, it would pose, but if to the <planet>, a human outfit scanner you could bring, alter it to suit our purposes, we will."`
+			choice
+				`	"Alright, I'll go there once I have an outfit scanner ready."`
+				`	"That sounds even more dangerous than what you had me do before. Sorry, but I'm not interested."`
+					decline
+			`	They thank you for your willingness to help, and tell you they will be waiting on the <destination>.`
+				accept
+	on complete
+		require "Outfit Scanner"
+
+mission "Heliarch Recon 2-B"
+	name "Silent Scan"
+	description "Now that the Heliarchs have altered your Outfit Scanner, head to <waypoints> and scan some Quarg ships there, then report back to <destination>."
+	landing
+	source "Ring of Wisdom"
+	waypoint "Enif"
+	to offer
+		has "Heliarch Recon 2-A: done"
+		not "Resistance: Questions: done"
+	on offer
+		conversation
+			`The Heliarch agents who spoke to you are ready with a team of engineers once you land. They remove the Outfit Scanner from your ship and take it to some restricted area. After about an hour, they come back, and while you can't see any visible changes to the outfit, they assure you it is better suited for scanning Quarg ships now.`
```

Comment:
Scanning equipment sensitivity is heavily dependent on design and component quality. They might be able to apply some software to see through interference or something, but if it comes to just scan accuracy they're going to have actually provide a coalition scanner, there's nothing they can do to your scanner that will help that.

Reply:
They could have swapped out the components and installed a program that prevents your scanner from using its full power on anything other than a quarg ship. Then when you return they swap the original components back in.

Reply:
It's just meant to be them gradually getting you to get more information for them.
You go with an Outfit Scanner and they do some tweaking on it, you go and get the data.
They want more data, so you're tasked with using an actual Scanning Module next.